analysis/import/SCMLogParser/src/test/resources/expected_git_numstat.json

```diff
@@ -140,7 +140,7 @@
                                     "attributes": {
                                       "number_of_authors": 2,
                                       "number_of_commits": 3,
-                                      "range_of_weeks_with_commits": 128,
+                                      "range_of_weeks_with_commits": 76,
```

Comment:
Why did this change?

Reply:
Wierd thing in PR https://github.com/MaibornWolff/codecharta/pull/2564, but nevertheless I had to fix a copy and paste mistake.